.golangci.yml

```diff
@@ -0,0 +1,14 @@
+linters-settings:
+  wsl:
+    allow-cuddle-declarations: true
+
+issues:
+  exclude-rules:
+    - linters: ["gocritic"]
+      text: "ifElseChain:"
+    - linters: ["gosec"]
+      text: "G204:"
+      path: util.go
+    # 1 is not a magic number. 3 is a magic number.
+    - linters: ["gomnd"]
```

Comment:
I am actually debating if we should just disable this linter. Do you see value? If so lets use the // nolint method to suppress where applicable.

Reply:
I dont have a strong feeling on it. It seems reasonable to not have hard coded numbers like 17.
But I just do not think that '1' is a hard coded magic number.

I'd suggest we leave this as it is. If its annoying we can get rid of it later.  At this point 0 and 1 are allowed.